src/components/Card/Card.js

```diff
@@ -41,32 +41,41 @@ function Card({
           )}
         </button>
       </div>
-      <div className='card-info' id={key}>
-        <h2>{showtime.theater}</h2>
-        <p>{dayjs(showtime.date).format("MMM. D YYYY | hh:mm A")}</p>
-
+     <div className='card-info' id={key}>
+        <p className='card-theaterName'>{showtime.theaterName}</p>
         {showtime.movies.length === 1 ? (
-          showtime.movies[0].director ? (
-            <p>
+          showtime.movies[0].director.trim() ? (
+            <div>
+            <p className='card-title'>
             {" "}
-            {showtime.movies[0].title}, {showtime.movies[0].year} | dir. by {" "}
-            {showtime.movies[0].director} 
+            {showtime.movies[0].title}
+            {showtime.movies[0].year ? `, ${showtime.movies[0].year}` : ""}
```

Comment:
Jen,
I really like the way you're using the ternary operator for your conditionals.  This is great!